components/datetime/src/neo.rs

```diff
@@ -357,9 +357,7 @@ where
     ///     )
     ///     .unwrap();
     ///
-    /// // the trait `GetField<AnyCalendarKind>`
-    /// // is not implemented for `icu::icu_calendar::Time`
```

Comment:
Issue, here and elsewhere: This comment is semantically important; it documents the error that happens when calling this function. It can help both learnability and discoverability when searching the docs for an error message. Why did you delete it?

Reply:
ok. It should be fixed to be whatever the current error is, and the error number should be added to the docs test specification.